prometheus-exporter/templates/deployment.yaml

```diff
@@ -1,51 +1,73 @@
-apiVersion: apps/v1
+apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
   name: {{ include "prometheus-exporter.fullname" . }}
   labels:
-    app.kubernetes.io/name: {{ include "prometheus-exporter.name" . }}
-    helm.sh/chart: {{ include "prometheus-exporter.chart" . }}
-    app.kubernetes.io/instance: {{ .Release.Name }}
-    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app: {{ include "prometheus-exporter.fullname" . }}
+    chart: {{ include "prometheus-exporter.chart" . }}
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
 spec:
-  replicas: {{ .Values.replicaCount }}
-  selector:
-    matchLabels:
-      app.kubernetes.io/name: {{ include "prometheus-exporter.name" . }}
-      app.kubernetes.io/instance: {{ .Release.Name }}
+  replicas: 1
```

Comment:
Why do you hardcode the replicas to 1? I think we should be able to scale the exporters if we want to.